documentation/UsingSDKWithWristUnits.md

```diff
@@ -0,0 +1,12 @@
+[Back to main page](../README.md)
+
+# Using SDK with Polar wrist units
+
+Polar wrist units such as Polar Ignite 3, Polar Pacer (Pro), Polar Grit X Pro 2 or Polar Vantage V3 support online streaming of some data types,
+altough some special user action is required to be able to start the streaming feature and get the data.
+
+> [!IMPORTANT]
+>
+> Although the namings might be confusing, enabling the SDK on the wrist unit is not starting [SDK Mode](./SdkModeExplained.md) with `startSdkMode()` function. SDK Mode is currently not supported in Polar watches.
+
+TODO: Add steps here
```

Comment:
TODO here, either fix or remove the text and chapter

Reply:
That was left here voluntarily as I don't know these steps,  but I think they should be added here as the whole process seems very unclear to the users (and even to me). I could do in this PR if someone gives me the needed info.

Reply:
maybe ask from the team who implemented the feature?

Reply:
Step by step guide added to this document